generate_adapter.py

```diff
@@ -18,7 +18,8 @@
 def main(
     prompt: str = "What food do lamas eat?",
     input: str = "",
-    checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
+    adapter_path: Optional[Path] = None,
+    pretrained_dir: Optional[Path] = None,
```

Comment:
Same here, see comment above. 
For evaluation purposes, it makes sense to be able to read different finetuned checkpoints from a separate directory.

Reply:
100% agreed